MelonLoader.Bootstrap/Logging/ColorRGB.cs

```diff
@@ -1,18 +1,540 @@
+    }
+
+    /// <summary>Creates a <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> structure from the specified 8-bit color values (red, green, and blue). The alpha value is implicitly 255 (fully opaque).</summary>
+    /// <param name="red">The red component value for the new <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" />.</param>
+    /// <param name="green">The green component value for the new <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" />.</param>
+    /// <param name="blue">The blue component value for the new <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" />.</param>
+    /// <returns>The <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> that this method creates.</returns>
+    public static ColorARGB FromArgb(byte red, byte green, byte blue)
+    {
+      return FromArgb(byte.MaxValue, red, green, blue);
+    }
+    
+    /// <summary>Tests whether two specified <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> structures are equivalent.</summary>
+    /// <param name="left">The <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> that is to the left of the equality operator.</param>
+    /// <param name="right">The <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> that is to the right of the equality operator.</param>
+    /// <returns>
+    /// <see langword="true" /> if the two <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> structures are equal; otherwise, <see langword="false" />.
+    /// </returns>
+    public static bool operator ==(ColorARGB left, ColorARGB right)
+    {
+        return left.value == right.value;
+    }
+
+    /// <summary>Tests whether two specified <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> structures are different.</summary>
+    /// <param name="left">The <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> that is to the left of the inequality operator.</param>
+    /// <param name="right">The <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> that is to the right of the inequality operator.</param>
+    /// <returns>
+    /// <see langword="true" /> if the two <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> structures are different; otherwise, <see langword="false" />.
+    /// </returns>
+    public static bool operator !=(ColorARGB left, ColorARGB right) => !(left == right);
+
+    /// <summary>Tests whether the specified object is a <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> structure and is equivalent to this <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> structure.</summary>
+    /// <param name="obj">The object to test.</param>
+    /// <returns>
+    /// <see langword="true" /> if <paramref name="obj" /> is a <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> structure equivalent to this <see cref="T:MelonLoader.Bootstrap.Logging.ColorARGB" /> structure; otherwise, <see langword="false" />.
+    /// </returns>
+    public override bool Equals(object obj)
```

Comment:
Please resolve this warning. If I'm not mistaken, even if the project isn't nullable, making the type nullable should work fine as long as the project uses the latest lang version.
If I'm wrong, do a manual suppression instead